pkg/service/system/creds.go

```diff
@@ -89,7 +89,7 @@ func (g *GithubPersonalAccessToken) GetAws() *AwsCredential {
 }
 
 func (g *GithubPersonalAccessToken) AsSecretValue() string {
-	return fmt.Sprintf("%s", g.Token)
+	return g.Token
```

Comment:
@NishanthSpShetty was this the way it was previously for a reason?

Reply:
no, had multiple fields to convert it to string, so to keep consistent it was kept as is.